proto/p4/v1/p4runtime.proto

```diff
@@ -457,6 +464,10 @@ message Replica {
     bytes port = 3;
   }
   uint32 instance = 2;
+  // List of backup replicas used as a fallback when the primary replica port
```

Comment:
"when the primary replica port goes down" -> "when the primary replica port (and all higher-preference backup replica ports) are down"